src/Nethermind/Nethermind.Facade/Proxy/Models/MultiCall/Error.cs

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+namespace Nethermind.Facade.Proxy.Models.MultiCall;
+
+public class Error
+{
+    public int Code { get; set; }
```

Comment:
You can use existing JSON RPC errors classes

Reply:
Will do.

Reply:
Sadly not really possible, as we are bound by the BlockchainBridge that is in Facade and we need the error in there... So it looks more likely we could bring the Error from RPC back into Facade...